<?xml version="1.0"?>

<launch>
    <!-- For Simulation -->
    <arg name="sim_mode" default="false" />
    <param name="/use_sim_time" value="$(arg sim_mode)"/>
    
    <!-- move base -->
    <node pkg="tianracer_navigation" type="multi_goals.py" respawn="false" name="multi_goals" output="screen">
        <!-- params for move_base -->
        <param name="goalListX" value="[11.3,3.5, -3.48]" />       
         <param name="goalListY" value="[-5.5,-15.7, -1.40]" />

        <param name="goalListZ" value="[-0.822, 0.754, 0.107" />
        <param name="goalListW" value="[0.57, 0.656, 0.994]" />
        <param name="map_frame" value="map" />
    </node>
    

</launch>
